internal/locale/locale.go

```diff
@@ -88,17 +91,21 @@ func getLocaleFlag() string {
 }
 
 // Set the active language to the given locale
-func Set(localeName string) {
+func Set(localeName string) error {
 	if !funk.Contains(Supported, localeName) {
-		fmt.Printf("Locale does not exist: %s\n", localeName)
-		exit(1)
-		return
+		return errs.New("Locale does not exist: %s", localeName)
 	}
 
 	translateFunction, _ = i18n.Tfunc(localeName)
 	_ = translateFunction
 
-	config.Get().Set("Locale", localeName)
+	cfg, err := config.Get()
```

Comment:
Should be fed in. I understand this is problematic since locale itself is "global". But we can set the cfg param inside this package from the main package.